newm-server/src/main/kotlin/io/newm/server/features/minting/MintingMessageReceiver.kt

```diff
@@ -177,6 +177,10 @@ class MintingMessageReceiver : SqsMessageReceiver {
                     if (!cardanoRepository.isMainnet()) {
                         // force a track into the “An error occurred” status when I attempt to distribute a song in the TestNet environment
                         if (song.title?.contains("[ForceError]", true) == true) {
+                            songRepository.updateSongMintingStatus(
```

Comment:
This doesn't seem like it's needed. It should be caught and db updated in the catch block below. Then it gets wrapped in a DistributeAndMintException and logged to Sentry.

Reply:
I agree with you, catch should update the status. 

Jonatan was able to track one of these songs yesterday, and he received this notification. 
But UI didn't show the correct status. I don't have access to DB to confirm status for that song. 

![image](https://github.com/projectNEWM/newm-server/assets/14129154/4d653cdf-7570-40ba-8c06-6cf9fae84e70)